x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/bulk_actions/route.ts

```diff
@@ -322,6 +323,23 @@ export const performBulkActionRoute = (
                 .filter((rule): rule is RuleAlertType => rule !== null);
               break;
             }
+
+            case BulkActionTypeEnum.backfill: {
+              const { backfilled, errors: bulkActionErrors } = await bulkScheduleBackfill({
+                rules,
+                isDryRun,
+                rulesClient,
+                mlAuthz,
+                backfillPayload: body.backfill,
+                experimentalFeatures: config.experimentalFeatures,
+              });
+              errors.push(...bulkActionErrors);
+              return buildBulkScheduleBackfillResponse(response, {
+                errors,
+                isDryRun,
```

Comment:
Building the response seems to be the same whether it is `dryRun` or not.

Reply:
done